Makefile

```diff
@@ -48,7 +48,8 @@ SRCS-y += lib/mailbox.c lib/net.c lib/flow.c lib/ipip.c \
 
 LDLIBS += $(LDIR) -Bstatic -lluajit-5.1 -Bdynamic -lm -lmnl
 CFLAGS += $(WERROR_FLAGS) -I${GATEKEEPER}/include -I/usr/local/include/luajit-2.0/
-EXTRA_CFLAGS += -O3 -g -Wfatal-errors -DALLOW_EXPERIMENTAL_API
+EXTRA_CFLAGS += -O3 -g -Wfatal-errors -DALLOW_EXPERIMENTAL_API \
+	-DLUA_BASE_DIR='"./lua"' -DGATEKEEPER_CONFIG_FILE='"gatekeeper_config.lua"'
```

Comment:
1. No. Otherwise, we will get errors: ``/home/qiaobin/gatekeeper/config/static.c: In function ‘config_gatekeeper’:``
``<command-line>:0:14: error: expected expression before ‘.’ token``
`` note: in expansion of macro ‘LUA_BASE_DIR’
    "%s/%s", LUA_BASE_DIR, GATEKEEPER_CONFIG_FILE);``

Reply:
1. Thank you for trying it out.